tags/views.py

```diff
@@ -505,14 +571,101 @@ def highlight(request):
   }
 
 '''
-Get all highlights for a page
+Add a highlight to a page
 '''
+@csrf_exempt
 @login_required
 @ajax_request
-def highlights(request):
+def highlight(request):
```

Comment:
confused. I see /highlights below. did you just swap the order of these two endpoints (/highlight and /highlights)?

Reply:
Yeah to make it more readable -  I structured the file so that certain functions were near each other